arm_neon is defined because the toolchain supports it.
however, we only want it on aarch64, not the 32-bit arm's where we don't support
it.
--
diff --git a/src/lib/OpenEXRCore/internal_zip.c b/src/lib/OpenEXRCore/internal_zip.c
index e829e6d..4f378db 100644
--- a/src/lib/OpenEXRCore/internal_zip.c
+++ b/src/lib/OpenEXRCore/internal_zip.c
@@ -24,7 +24,7 @@
 #    define IMF_HAVE_SSE4_1 1
 #    include <smmintrin.h>
 #endif
-#if defined(__ARM_NEON)
+#if defined(__ARM_NEON) && defined(__aarch64__)
 #    define IMF_HAVE_NEON 1
 #    include <arm_neon.h>
 #endif
